media/css/firefox/switch.scss

```diff
@@ -5,48 +5,61 @@
 $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 @import '../../protocol/css/includes/lib';
-@import '../../protocol/css/components/hero';
+@import '../../protocol/css/components/split';
+@import '../../protocol/css/components/logos/wordmark';
+@import '../../protocol/css/components/logos/wordmark-product-firefox';
 
 
-.t-switch.mzp-c-hero {
+.mzp-c-split {
+    overflow: visible;
```

Comment:
I think this is causing a horizontal scroll. Maybe specify overflow-y?
![Screen Shot 2021-08-24 at 2 49 02 PM](https://user-images.githubusercontent.com/19650432/130632968-a76cb419-8d6b-4c2a-affe-dff4f92b68d9.png)

Reply:
Ah for some reason I wasn't getting this issue when I first worked on it, and I was only able to replicate it briefly (with several restarts of the server). I'll push the changes again and will have you (or anyone on the team) test it to see if it's still wonky! Thanks for the catch :)

Reply:
Interesting, no I don't see it on the demo server or locally now.